.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
seems we're missing a rule in `.gitignore` to exclude golang configuration files :)

Reply:
Oops will add and ignore the .idea file.